### DISCUSSION TITLE
Hold bind and run continuously

### DISCUSSION THREAD

**Discussion by xVermillionx** [hyprwm/Hyprland#295] [Q&A] [answered]
I am trying to setup key binds that execute some command continuously like for example increasing the volume or the brightness by holding a certain binding.

I was not able to find any documentation or discussion on this problem, am I missing something or is it currently not possible to set this up?

**UPDATE:**
Using the e flag on bind (binde=...) binds are executed continuously on hold
Wiki: https://github.com/hyprwm/Hyprland/wiki/Advanced-config#bind-flags

**Comment by vaxerski** [comment, marked as answer]
not possible

**Comment by fufexan** [comment]
At least for brightness, `light -A 5` / `light -U 5` works repeatedly when I hold the key.
